codecs/jxl/Makefile

```
@@ -78,9 +78,12 @@ $(CODEC_MT_SIMD_BUILD_DIR)/Makefile: CXXFLAGS+=-msimd128
 	-B $(@D) \
 	$(<D)
 
-$(CODEC_DIR)/CMakeLists.txt:
-	mkdir -p $(@D)
-	git clone $(CODEC_URL) --recursive -j`nproc` --depth 1 --branch $(CODEC_VERSION) $(@D)
+$(CODEC_DIR)/CMakeLists.txt: 
+	rm -rf $(@D)
+	git init $(@D)
+	git -C $(@D) fetch $(CODEC_URL) $(CODEC_VERSION) --depth 1
+	git -C $(@D) checkout FETCH_HEAD
+	git -C $(@D) submodule update --init --recursive --jobs `nproc`
```

Found one more optimisation that was in my fork but not here:

```suggestion
	git -C $(@D) submodule update --init --depth 1 --recursive --jobs `nproc`
```